airflow-core/src/airflow/jobs/triggerer_job_runner.py

```diff
@@ -419,6 +429,8 @@ def _handle_request(self, msg: ToTriggerSupervisor, log: FilteringBoundLogger, r
                 dump_opts = {"exclude_unset": True, "by_alias": True}
             else:
                 resp = conn
+        elif isinstance(msg, DeleteVariable):
+            self.client.variables.delete(msg.key)
```

Comment:
```suggestion
            resp = self.client.variables.delete(msg.key)
```

I request you to follow the shared reference (same mechanism used in supervisor & dag processor). The response catch here does something and is needed.

Reply:
@amoghrajesh Thanks for calling that out. I thought it would be the same for all OkResponse types. 

I have made the requested change, so it should be good to merge if nothing else is needed.

Reply:
All good, thanks!